dist/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <title>To Do list</title>
+<script defer src="main.js"></script></head>
+<body>
+    <section class="pm-font" id="todo-list-content">
+        <h1 class="pm-font pm-border icon-set">
+            Today's To Do
+            <a class="action-icon" href="#">
+                <span class="material-symbols-outlined">
+                    sync
+                </span>
+            </a>
+        </h1>
```

Comment:
Kindly, remove the anchor tag from the h1 tag for better semantic understanding😉